src/style.css

```diff
@@ -0,0 +1,116 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+html {
+  font-family: Avenir, sans-serif;
+  font-style: normal;
+}
+
+.search {
+  height: 70px;
+  margin: 20px 0;
+}
+
+.searchSmall {
+  height: 42px;
+}
+
+.img {
+  width: 19px;
+}
+
+.imgSmall {
+  width: 11px;
+}
+
+.input {
+  box-sizing: border-box;
+  width: 100%;
+  outline: none;
+  padding: 0 0 0 62px;
+  font-family: inherit;
+  height: 70px;
+  font-size: 16px;
+  font-weight: 800;
+  line-height: 22px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
```

Comment:
Please make variables for colors you use several times and give clear names for these variables